src/libraries/System.IO.FileSystem.DriveInfo/src/System/IO/DriveInfo.UnixOrBrowser.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Security;
+
+namespace System.IO
+{
+    public sealed partial class DriveInfo
+    {
+        public static DriveInfo[] GetDrives()
+        {
+            string[] mountPoints = GetMountPoints();
+            DriveInfo[] info = new DriveInfo[mountPoints.Length];
+            for (int i = 0; i < info.Length; i++)
+            {
+                info[i] = new DriveInfo(mountPoints[i]);
+            }
+
+            return info;
+        }
+
+        private static string NormalizeDriveName(string driveName)
+        {
+            if (driveName.Contains("\0")) // string.Contains(char) is .NetCore2.1+ specific
```

Comment:
Incidentally I think this comment is out of date and `if (driveName.Contains('\0'))` ought to compile.

Reply:
Yeah looks like it but since this is preexisting code I'll leave the cleanup to later on :)